src/usr/local/www/classes/Form/Checkbox.class.php

```diff
@@ -56,9 +56,9 @@ protected function _getInput()
 	{
 		$input = parent::_getInput();
 
-		if (empty($this->_description))
-			return '<label class="chkboxlbl">'. $input .'</label>';
+		if (!empty($this->_description) || is_numeric($this->_description))
```

Comment:
@phil-davis - Due to exact same idiocy with ```empty()```, there's ```if ($foo != '') { }``` used in zillions of places.